core/src/main/kotlin/org/evomaster/core/search/algorithms/CroReactor.kt

```diff
@@ -0,0 +1,152 @@
+package org.evomaster.core.search.algorithms
+
+import org.evomaster.core.EMConfig
+import org.evomaster.core.search.Individual
+import org.evomaster.core.search.algorithms.wts.WtsEvalIndividual
+import org.evomaster.core.search.service.Randomness
+
+class CroReactor<T>(
```

Comment:
what is a "reactor" in this context? also, if there is going to be only one instance, should consider to use a singleton service

Reply:
also, any special design reason why this `CroReactor` is separated from `CroAlgorithm`?